### PR TITLE
Fixed the build

### PR DESCRIPTION
**Pull request by dlardo**
- Image wasn't running due to this error: `Failed to open dedicated.so (libstdc++.so.6: cannot open shared object file: No such file or directory)`
- Fixed by adding in `apt-get install lib32stdc++6`
- Added more ENV var support for the make file
- Dockerfile best practices

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -20,8 +30,19 @@ run:
...
 		-e "STEAM_ACCOUNT=$(STEAM_ACCOUNT)" \
-		$(IMAGE_NAME)
\ No newline at end of file
+		-e "TICKRATE=$(TICKRATE)" \
+		-e "GAME_TYPE=$(GAME_TYPE)" \
+		-e "GAME_MODE=$(GAME_MODE)" \
+		-e "MAP=$(MAP)" \
+		-e "MAPGROUP=$(MAPGROUP)" \
+		-e "MAXPLAYERS=$(MAXPLAYERS)" \
+		--name $(CONTAINER_NAME)" \
```
Comment (kaimallea) [inline Makefile]: it looks like an extra quote at the end of this line that needs to be removed. it's causing this error:

```bash
/bin/bash: -c: line 0: unexpected EOF while looking for matching `"'
/bin/bash: -c: line 1: syntax error: unexpected end of file
make: *** [run] Error 2
```

**Comment by kaimallea** [comment]
hey, this is great! 🚀 thank you so much for contribution! it works great; there's just one minor extra character that's breaking the `make run` command (see review comment).

**Comment by dlardo** [comment]
Wow, no idea how I botched that up. Should be fixed now?

**Comment by kaimallea** [comment]
@dlardo you're awesome, thanks again for fixing the build!

**Pull request merged**